common/src/main/java/com/microsoft/identity/common/adal/internal/AuthenticationConstants.java

```diff
@@ -529,6 +529,23 @@ public static final class AAD {
         public static final String APP_VERSION = "x-app-ver";
     }
 
+    /**
+     * Represents the constants value for the DUNA flow.
+     */
+    @NoArgsConstructor(access = AccessLevel.PRIVATE)
+    public static final class DUNA {
+
+        /**
+         * String Query parameter key for the session token.
+         */
+        public static final String SESSION_TOKEN = "session_token";
+
+        /**
+         * String Query parameter key for the endpoint.
+         */
+        public static final String ENDPOINT = "endpoint";
```

Comment:
As mentioned to you in chat, this is now called `action_uri`

Reply:
Your are to fast my friend